prime-router/src/test/kotlin/fhirengine/engine/fhirRouterTests/RoutingTests.kt

```diff
@@ -2,6 +2,7 @@ package gov.cdc.prime.router.fhirengine.engine.fhirRouterTests
 
 import assertk.assertThat
 import assertk.assertions.isEqualTo
+import assertk.assertions.isGreaterThan
```

Comment:
Just noticed that this is an unused import, which make ktlint complain, try to fix this before merging!

Reply:
fixed